frontend/packages/operator-lifecycle-manager/src/components/operand.tsx

```diff
@@ -466,6 +466,10 @@ export const OperandDetails = connectToModel((props: OperandDetailsProps) => {
   );
 });
 
+const ResourcesTab = (resourceProps) => (
```

Comment:
nit 
```suggestion
const ResourcesTab = resourceProps => (
```

Reply:
I think the suggestion will fail prettier